certification/internal/policy/container/base_on_ubi_test.go

```diff
@@ -29,28 +31,27 @@ func labelsForUbiCheck(label string, bad bool) map[string]string {
 	return labels
 }
 
-func goodLabelsConfigFile() (*cranev1.ConfigFile, error) {
+func LabelsConfigFile() (*cranev1.ConfigFile, error) {
 	return &cranev1.ConfigFile{
 		Config: cranev1.Config{
-			Labels: labelsForUbiCheck("ubi8-container", false),
+			Labels: labelsForUbiCheck("ubi-container", false),
```

Comment:
This whole func is no longer needed.

Reply:
was getting a null pointer so left it... but all of that is probably overkill just need a subset will try to remove most of it..

Reply:
removed